mits/site_sections/mits_admin.py

```diff
@@ -9,8 +9,96 @@ def index(self, session, message=''):
             'teams': session.mits_teams()
         }
 
+    def create_new_application(self):
+        cherrypy.session.pop('mits_team_id', None)
+        raise HTTPRedirect('../mits_applications/team')
+
     def team(self, session, id, message=''):
         return {
             'message': message,
             'team': session.mits_team(id)
         }
+
+    def set_status(self, session, id, status=None, confirmed=False, csrf_token=None, return_to='index', message=''):
```

Comment:
The way this page handlers works is that if we try to set the status, the system checks for a couple of warning cases:
- if the status was already set to something other than PENDING
- if the application isn't complete
- if there are other teams with the same name as this one which haven't yet been deleted as duplicates

If none of those things are the case, then we just perform the save.  If any of them are true, then we instead throw up a warning page, which re-submits the form with a ``confirmed`` parameter.